src/busco/config.vsh.yaml

```diff
@@ -0,0 +1,210 @@
+functionality:
+  name: busco
+  description: Assessment of genome assembly and annotation completeness with single copy orthologs
+  info:
+    keywords: [Genome assembly, quality control]
+    homepage: https://busco.ezlab.org/
+    documentation: https://busco.ezlab.org/busco_userguide.html
+    repository: https://gitlab.com/ezlab/busco
+    reference: "10.1007/978-1-4939-9173-0_14"
+    licence: MIT
+  argument_groups:
+    - name: Inputs
+      arguments:
+        - name: --input
+          alternatives: ["-i"]
+          type: file
+          description: |
+            Input fasta file or directory containing input fasta files to analyse. Fasta files can either be a nucleotide or protein fasta file, depending on the BUSCO mode.
+          required: true
+          example: file.fasta
+        - name: --mode
+          alternatives: ["-m"]
+          type: string
+          choices: ["protein", "genome", "transcriptome"]
+          required: true
+          description: |
+            Busco assesment mode
+          example: protein
+        - name: --lineage_dataset
+          alternatives: ["-l"]
+          type: string
+          required: false
+          description: |
+            Specify a BUSCO lineage dataset that is most closely related to the assembly or gene set being assessed. 
+            The full list of available datasets can be viewed using "busco --list-datasets".
+            When unsure, the "--auto-lineage" flag can be set to automatically find the optimal lineage path.
+            Requested datasets will automatically be downloaded if not already present in the download folder.
+          example: stramenopiles_odb10
+
+    - name: Outputs
+      arguments:
+        - name: --output_dir
+          alternatives: ["-o"]
+          required: true
+          direction: output
+          type: file
+          description: |
+            Path to output directory for publishing BUSCO results
+          example: output
+        - name: --output_prefix
+          type: string
+          required: false
+          description: |
+            Name of the analysis run, output folders and files will be labeled with this name. if not specified the output will be the input file name."
+          example: busco_protein
+
+    - name: Resource and Run Settings
+      arguments:
+        - name: --force
+          type: boolean_true
+          description: |
+            Force rewriting of existing files. Must be used when output files with the provided name already exist.
+        - name: --offline
+          type: boolean_true
+          description: |
+            In offline mode BUSCO will not attempt to download files. Ensure all required dataset files are already downloaded and available.
```

Comment:
shall i leave it out?

Reply:
Up to you to decide! Either leave it out, or implement other components that allow users to first list and download the necessary datasets and then pass it as an input for this component ^^

Personally I'd leave it out -- if anybody ever needs the proposed functionality, it can be added to the repo.